py/scripts/template-version.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+VERSION_FILE="src/braintrust/version.py"
+
+GIT_COMMIT=$(git rev-parse HEAD)
+
+sed_inplace() {
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        sed -i '' "$@"
+    else
+        sed -i "$@"
```

Comment:
gnu sed is best sed
![image](https://github.com/user-attachments/assets/5022e702-920b-47b2-b55f-630dabaa5641)